dask/array/tests/test_optimization.py

```diff
@@ -117,13 +116,6 @@ def test_hard_fuse_slice_cases():
     assert optimize_slices(dsk) == {'x': (getarray, 'x', (None, 5))}
 
 
-def test_dont_fuse_different_slices():
```

Comment:
This test could be retained if there was a way to return to the original behavior, perhaps something like the following:

    y = x.rechunk((1, 10), threshold=10000)